lib/oga/xml/document.rb

```diff
@@ -57,6 +57,12 @@ def root_node
         self
       end
 
+      # Returns the top level element in the document.
+
```

Comment:
Is allowing multiple roots intentional?  Mightn't it be better to either throw an exception on adding a second root element (since that's not allowed by the XML spec)?